Srl 694  demand letter

reshmai opened the pull request.

Commit:
Js file added for change date format as per functionality in webform message

Commit:
Merge branch 'master' into SRL-694--demand-letter

melwong-jcc reviewed (changes requested):
For better function context, either name the file more specific (example: calculateCaseTypeDateFinal.js) and/or put comments in file.

Other factors:

- What is the final date being calculated for? (example: calculateCaseTypeDateFinal.js)
- Any way to make the max duration configurable for other cases? Or too complicated to cover multiple cases? (example: calculateDateFinal.js)

melwong-jcc commented:
Also, not sure what kind of JQuery support will be left in future Drupal versions so we should consider JS that does not rely on a third-party library for the next one. Don't worry about that for this one.

Commit:
Added changes into date calculation code as per review

reshmai commented:
> Also, not sure what kind of JQuery support will be left in future Drupal versions so we should consider JS that does not rely on a third-party library for the next one. Don't worry about that for this one.



> For better function context, either name the file more specific (example: calculateCaseTypeDateFinal.js) and/or put comments in file.
> 
> Other factors:
> 
> * What is the final date being calculated for? (example: calculateCaseTypeDateFinal.js)
> * Any way to make the max duration configurable for other cases? Or too complicated to cover multiple cases? (example: calculateDateFinal.js)

I have changed a filename and also added comment in js file.

Commit:
Name has been changed into js file
